src/content/app/tools/blast/utils/tests/blastInputParser.test.ts

```diff
@@ -57,10 +57,5 @@ describe('parseBlastInput', () => {
       const input = readFile('./input-sequence-examples/006.txt');
       expect(parseBlastInput(input)).toMatchSnapshot();
     });
-
-    it('removes numbers from numbered sequence lines', () => {
-      const input = readFile('./input-sequence-examples/007.txt');
-      expect(parseBlastInput(input)).toMatchSnapshot();
-    });
```

Comment:
Should either:
- change the test to "keeps numbers in numbered sequence lines", and update the snapshot, or
- delete the snapshot and the input text example

The first option is probably more preferable.

Reply:
@azangru i have updated the test to say "does not remove non-letter characters", i think this way it makes it clearer about the decision we took and also what we are testing. 

Regarding your comment about `reformatiing the input to 60-character lines,...` i am not sure i understand what you meant. Where are we reformatting the input to 60-character lines?

Reply:
> i am not sure i understand what you meant

Add the content of the 007.txt file into the input box, and see what happens:

![image](https://user-images.githubusercontent.com/6834224/155988253-121cf5d7-6b5c-456c-95e2-341712bfa1e2.png)